notebooks/02_model/surprise_svd_deep_dive.ipynb

```diff
@@ -748,20 +732,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## References\n",
+    "## 4. To Read More...\n",
     "\n",
-    "1. Ruslan Salakhutdinov and Andriy Mnih. Probabilistic matrix factorization. 2008. URL: http://papers.nips.cc/paper/3208-probabilistic-matrix-factorization.pdf\n",
-    "2. Yehuda Koren, Robert Bell, and Chris Volinsky. Matrix factorization techniques for recommender systems. 2009.\n",
-    "3. Francesco Ricci, Lior Rokach, Bracha Shapira, and Paul B. Kantor. Recommender Systems Handbook. 1st edition, 2010."
+    "[Koren, Bell and Volinksy's paper in 2009](https://ieeexplore.ieee.org/document/5197422) provides a good overview of matrix factorization methods in recommendations.  Nicolas Hug, developer of the Surprise library and the author of the first versions of this notebook, has a nice 4-part [blog post](http://nicolas-hug.com/blog/matrix_facto_1) about matrix factorization and specifically SVD."
```

Comment:
In the notebooks [template](https://github.com/Microsoft/Recommenders/blob/master/notebooks/template.ipynb) we have created earlier, it is encouraged to add a reference section where a list of use learning materials mentioned in the notebook are provided to the reader. If needed, we can modify this section to be the one as shown in this notebook - in addition to the reference itself, some details about "why" it is cited is also provided. What do you (@wutaomsft @miguelgfierro) think?

Reply:
Yes, I think Reference section can be quite valuable.  The only thoughts I have are that 
1. In blogs, it is more popular to link the references directly  rather than point to the bibliography at the end of the blog.  In Wikipedia it is the other way around. In notebooks, the two approaches have their own pros and cons.  Direct link reduce context switching and I find it is instantly satisfying; On the other hand, if a particular paper is to be referenced multiple times, it is quite easy to use bibliography.  In this PR I used the first approach but used a very descriptive section about related papers in the hope that people who are interested in knowing more can get a little bit of context/history.
2. I think any papers appearing in References should be referenced (or discussed) at least once in the notebook; otherwise it is not clear to the reader why it is relevant.  I removed a couple of papers based on this criteria but think they could be added back if we can introduce them properly.  
Any thoughts/comments on the above?

Reply:
Ah I see. Agree. This makes a lot of sense!

Reply:
agree